src/Hosting.CommandLine/McMaster.Extensions.Hosting.CommandLine.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>Latest</LangVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>true</IsPackable>
+    <Description>Generic Host integration for command-line parsing API.</Description>
+    <PackageDescription>Command-line parsing API and utilities for console applications.
```

Comment:
Can you update this description to describe this package?

Reply:
I _basically_ copied the `.csproj` from the main project.  I have no idea what `PackageDescription` is supposed to be (it doesn't appear in [the documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#description)) so I didn't modify it.  The `Description` already contains a good description for this project, so I just [removed `ProjectDescription`](https://github.com/natemcmaster/CommandLineUtils/pull/167/commits/bc78fbef60130679176129b3886a3a53292da75c).  That work?

Reply:
Yup, that's fine.

The different, by the way, is that `Description` is used for `System.Reflection.AssemblyDescriptionAttribute`, and `PackageDescription` is used for the NuGet package. `PackageDescription` defaults to `Description`.